cmd/core/app/server.go

```diff
@@ -177,12 +176,6 @@ func run(ctx context.Context, s *options.CoreOptions) error {
 		return err
 	}
 
-	dm, err := discoverymapper.New(mgr.GetConfig())
-	if err != nil {
-		klog.ErrorS(err, "Failed to create CRD discovery client")
-		return err
-	}
-	s.ControllerArgs.DiscoveryMapper = dm
 	pd, err := packages.NewPackageDiscover(mgr.GetConfig())
```

Comment:
Maybe we can deprecate `packageDiscover` as well.

Reply:
Yes. I would like to do so as well. Maybe in next PR.